neuvector-scanner.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: neuvector-scanner
+  version: 0_git20240404
+  epoch: 0
+  description: NeuVector vulnerability scanner for the SUSE NeuVector Container Security Platform
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    CGO_ENABLED: "1"
```

Comment:
just checking-in, was it intentional? Does project use `CGO_ENABLED=1`?

Reply:
Yes. The project requires it, for the build.